src/components/navBar/navBar.stories.tsx

```diff
@@ -25,20 +64,111 @@ export default meta
 const Logo = () => {
   return (
     <Box sx={{ display: 'flex' }}>
-      <Typography sx={{ color: 'secondary.main' }}>RUSTIC&nbsp;</Typography>
-      <Typography>UI</Typography>
+      <Typography sx={{ color: 'secondary.main', fontWeight: 700 }}>
+        RUSTIC&nbsp;
+      </Typography>
+      <Typography sx={{ fontWeight: 700 }}>UI</Typography>
     </Box>
   )
 }
 
+const NotificationsIcon = () => {
+  const theme = useTheme()
+  return (
+    <IconButton color="primary" sx={{ p: 0 }}>
```

Comment:
Do we want to add `IconButton` to the `navBar` component instead?

Reply:
I'm not sure about this since people could put possibly a search bar at the top as per the design use case.